RuleDocumentation/UseIdenticalMandatoryParametersForDSC.md

```diff
@@ -4,13 +4,24 @@
 
 ## Description
 
-The `Get-TargetResource`, `Test-TargetResource` and `Set-TargetResource` functions of DSC Resource must have the same mandatory parameters.
+For script based DSC resources the `Get-TargetResource`, `Test-TargetResource` and `Set-TargetResource` functions must have identical mandatory parameters that are also the keys in the corresponding `mof` file.
```

Comment:
Yes - except the last point is a little misleading. Set and Test functions should have _all identical parameters_, not just mandatory.

Also the first point should read: If a parameter is declared as mandatory in _any of Get/Set/Test_ then it should be a mandatory parameter in _all of Get/Set/Test_